client/notification.go

```diff
@@ -30,9 +31,10 @@ type NotificationCreatePayloadWith struct {
 }
 
 type NotificationUpdatePayload struct {
-	Name  string           `json:"name,omitempty"`
-	Type  NotificationType `json:"type,omitempty"`
-	Value string           `json:"value,omitempty"`
+	Name          string           `json:"name,omitempty"`
+	Type          NotificationType `json:"type,omitempty"`
+	Value         string           `json:"value,omitempty"`
+	WebhookSecret **string         `json:"webhookSecret,omitempty" tfschema:"-"`
```

Comment:
pointer to pointer allows me to support 3 modes:
1. passing a string "....". (modify the secret).
2. passing a null. (remove the secret)
3. omitting the field (no change)